.ci/test_r_package.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# set up R environment
+R_LIB_PATH=~/Rlib
+mkdir -p $R_LIB_PATH
+echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
+echo 'options(repos = "https://cran.rstudio.com")' > ${HOME}/.Rprofile
+export PATH="$R_LIB_PATH/R/bin:$PATH"
+
+# installing precompiled R for Ubuntu
+# https://cran.r-project.org/bin/linux/ubuntu/#installation
+# adding steps from https://stackoverflow.com/a/56378217/3986677 to get latest version
+if [[ $OS_NAME == "linux" ]]; then
+    sudo apt-get update
+    sudo add-apt-repository \
+        "deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/"
+    sudo apt-key adv \
+        --keyserver keyserver.ubuntu.com \
+        --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+    sudo apt-get update
+    sudo apt-get install \
+        -y \
+            r-base-dev=${R_LINUX_VERSION} \
+            texlive-latex-recommended \
+            texlive-fonts-recommended \
+            texlive-fonts-extra \
+            qpdf \
+            || exit -1
+fi
+
+# Installing R precompiled for Mac OS 10.11 or higher
+if [[ $OS_NAME == "macos" ]]; then
+    wget https://cran.r-project.org/bin/macosx/R-${R_MAC_VERSION}.pkg -O R.pkg
+    sudo installer \
+        -verbose \
```

Comment:
Is it really needed?

Reply:
I always like verbose logs for the purpose of CI, but I've turned this off to be consistent with how the rest of `test.sh` is structured

Reply:
Ah, I see! But at some point, with high logging level logs become so huge and in consequence poorly readable that cannot be rendered at CI web interface and you cannot find needed info quickly 😢 .